dev-libs/elfutils/files/elfutils-0.191-avoid-overriding-libcxx-system-header.patch

```diff
@@ -0,0 +1,28 @@
+https://bugs.gentoo.org/925241
+
+Replace -I with -iquote to avoid overriding stack system header from libcxx-18
```

Comment:
Can you send this upstream? I really don't want to carry this forever

Reply:
Id imagine there are better ways to handle this that wouldn't require hacking around -I., but it would reconfiguring the build system to avoid having built binaries in include paths.

So I feel better making a bug and see about how upstream feels about it first.

Reply:
ok, maybe ping mjw in #elfutils if you want to get his vibe first too (he might just say file a bug ofc, but i mean the option is there if you want). he's lovely

Reply:
ping